### ISSUE TITLE
History of SGLT2i medication use

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10496]
**Preferred term label:**
History of SGLT2i medication use

**Synonyms**

**Definition (free text, please give PubMed ID)**
Sodium glucose transport inhibitors; dapagliflozin; empagliflozin; canagliflozin; sglt-i; SGLT2; bexagliflozin; ertugliflozin; sotagliflozin; jardiance; invokana; forxiga; brenzavvy; steglatro; inpefa

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
SGLT-2i induced diabetic ketoacidosis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000949
- label: Sodium glucose transport inhibitor exposure
- parents: [Medication history (HP:4000112)](https://hpo.jax.org/app/browse/term/HP:4000112)
- pmid: 
- def: History of having been treateed with a sodium glucose transport inhibitors such as dapagliflozin, empagliflozin, canagliflozin, sglt-i, SGLT2, bexagliflozin, ertugliflozin, sotagliflozin, jardiance, invokana, forxiga, brenzavvy, steglatro, inpefa.
- comment: May be a risk factor for SGLT-2i induced diabetic ketoacidosis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10496
- synonyms: SGLT2i exposure(NONE)
